packages/react-card/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fluentui/react-cards",
+  "name": "@fluentui/react-card",
```

Comment:
Let's make this package private and not publish it, it would mean removing any examples in `react examples`

Since we are starting the `react-card` package from a clean slate let's only publish smth that is somewhat usable as an alpha

Reply:
Do you think we should also change the version to `9.0.0-alpha.0`, mainly to make sure that doesn't get missed later?

Reply:
Hmm actually having the examples still present within react-examples is causing the build to fail, since react-card is now missing from the dependencies. 

So maybe you should also go ahead and run the dx migration script? `yarn nx workspace-generator migrate-converged-pkg --name='@fluentui/react-card'`

Reply:
Actually never mind, this one still has v8 deps, so it can't be migrated. Maybe just delete the examples? (If someone wants to reference them later, they can find them in the history.) And leave the version as-is for now.

Reply:
I would copy `react-cards` back into v8 first, so that `react-examples` can use v8 instead. Then you don't have to delete the stories there

Reply:
The stories were for the converged-WIP Card, not the v8 Card. So that doesn't make sense. If we delete them, it's still possible to get them back from history if someone needs to reference them later.